events/conftest.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from django.utils import timezone
+from django.utils.datetime_safe import datetime
+from events.class_models.event_models import Colors
+from events.class_tests.create_events import TITLE
+from reminders.class_models.reminder_models import ReminderType
+
+
+# define constants
+def pytest_configure():
+    pytest.invalid_method = "undefined"
+    pytest.current_date = datetime.now()
+    pytest.valid_method = ReminderType.EMAIL
+    pytest.message = "Joined Meeting in 50 minutes"
+    pytest.valid_date_time = datetime(
+        pytest.current_date.year, pytest.current_date.month, pytest.current_date.day + 1, 12, 0, 0, tzinfo=timezone.utc
+    )
+    pytest.invalid_date_time = datetime(
+        pytest.current_date.year - 1, pytest.current_date.month, pytest.current_date.day, 12, 0, 0, tzinfo=timezone.utc
+    )
```

Comment:
These variables are not relevant to your tests and are not there for used by your functions
Please remove them from this PR,
In a later PR we will add conftest to all the apps and create the relevant variables if needed

Reply:
Actually, these variables are in use in my tests.